### ISSUE TITLE
v1.17.15 release artifacts versions are reported as v1.17.16-rc.0

### ISSUE THREAD

**Issue by ialidzhikov** [kubernetes/kubernetes#97241]
/kind bug
/kind regression
/sig release

**What happened**:
```
$ docker run k8s.gcr.io/hyperkube:v1.17.15 kubelet --version
Kubernetes v1.17.16-rc.0

$ docker run k8s.gcr.io/hyperkube:v1.17.15 kube-apiserver --version
Kubernetes v1.17.16-rc.0

$ docker run k8s.gcr.io/hyperkube:v1.17.15 kube-controller-manager --version
Kubernetes v1.17.16-rc.0

$ docker run k8s.gcr.io/kube-apiserver:v1.17.15 /usr/local/bin/kube-apiserver --version
Kubernetes v1.17.16-rc.0

$ docker run k8s.gcr.io/kube-controller-manager:v1.17.15 /usr/local/bin/kube-controller-manager --version
Kubernetes v1.17.16-rc.0
```

And this results in
```
$ k get no
NAME                     STATUS   ROLES    AGE   VERSION
cpu-worker-84c9f-54bg6   Ready    <none>   12m   v1.17.16-rc.0
```

**What you expected to happen**:

The version to be reported as non `-rc.0`. As it is done for all hyperkube releases before.
```
$ docker run k8s.gcr.io/hyperkube:v1.17.14 kubelet --version
Kubernetes v1.17.14
```

**How to reproduce it (as minimally and precisely as possible)**:
See above

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by Eddman** [comment]
Just installed using `kubeadm` into our staging cluster and see the same :smile:

**Comment by ialidzhikov** [comment]
This is not the first time we see such GitVersion mismatch - see https://groups.google.com/g/kubernetes-dev/c/Mhpx-loSBns/m/txJpuD9_DQAJ. As I commented in https://github.com/kubernetes/release/issues/1020#issuecomment-575390803, such GitVersion mismatch is breaking any existing version comparison logic.

**Comment by neolit123** [comment]
@kubernetes/release-engineering

**Comment by oldthreefeng** [comment]
does this really matter to an exsit cluster?  @neolit123

**Comment by ialidzhikov** [comment]
@oldthreefeng, see https://github.com/kubernetes/kubernetes/issues/97241#issuecomment-743265287

**Comment by ialidzhikov** [comment]
@justaugustus , is there any update on this one?

**Comment by hasheddan** [comment]
@ialidzhikov apologies for the delay here! Please see https://github.com/kubernetes/kubernetes/issues/97178 for more info

**Comment by ialidzhikov** [comment]
@hasheddan , few questions from my side:
1. Are v1.19.5 and v1.18.13 ready for production usage? I see that only v1.17.15 is affected by the GitVersion issue. Can you confirm
2. Sorry, my english is not good enough - what https://github.com/kubernetes/kubernetes/issues/97178#issuecomment-747572808 means? Will be a no-op v1.17.6 release to fix the GitVersion issue or the plan is simply to wait until the next patch release cycle?

**Comment by hasheddan** [comment]
@ialidzhikov all versions hit the issue of both the release and rc tags being on the same commit so we will be running accelerated patch releases (i.e. not on the regular schedule, but not necessarily a no-op because cherry picks have been merged to the release branches since the last round of releases) for each.

**Comment by ialidzhikov** [comment]
It seems that the issue is resolved with the v1.17.6 release.

/close
